gulpfile.js

```diff
@@ -957,7 +964,13 @@ function dist() {
         if (argv.fortesting) {
           return enableLocalTesting(minifiedRuntimeTarget).then(() => {
             if (!argv.single_pass) {
-              return enableLocalTesting(minifiedRuntimeEsmTarget);
+              return enableLocalTesting(minifiedRuntimeEsmTarget)
```

Comment:
could we do Promise.all on these instead?

```js
return Promise.all([enableLocalTesting(target1), enableLocalTesting(target2), enableLocalTesting(target3)]) 
```

Reply:
Will do in a future refactor.